Formula/superlu.rb

```diff
@@ -19,25 +19,21 @@ class Superlu < Formula
     sha256 "f2038e0b4edb755631cc4f9b42dc362996d8161fa9aad306a412c7e8ff39d9f8" => :sierra
   end
 
+  depends_on "cmake" => :build
   depends_on "gcc"
   depends_on "openblas"
 
   def install
-    ENV.deparallelize
-    cp "MAKE_INC/make.mac-x", "./make.inc"
-
-    args = ["SuperLUroot=#{buildpath}",
-            "SUPERLULIB=$(SuperLUroot)/lib/libsuperlu.a",
-            "CC=#{ENV.cc}",
-            "BLASLIB=-L#{Formula["openblas"].opt_lib} -lopenblas"]
-
-    system "make", "lib", *args
-    cd "EXAMPLE" do
-      system "make", *args
+    args = std_cmake_args + %W[
+      -Denable_internal_blaslib=NO
+      -DTPL_BLAS_LIBRARIES=#{Formula["openblas"].opt_lib}/libopenblas.dylib
```

Comment:
Can you use the `shared_libraries` DSL here, to make this work on linux too?

Reply:
Sure, just not 100% sure how that works. 

Does that just involve `s/libopenblas.dylib/#{shared_library("libopenblas")}`?

Reply:
Exactly. This will expand to the right extension based on the platform.

Reply:
`git push -f` is taking ages, but the commits should be on their way soon.